R/server.R

```diff
@@ -20,45 +21,109 @@ shinyServer(function(input, output, session) {
     observeEvent(input$upload_gs,{
         req(input$upload_gs)
         tryCatch({
-            uploaded_data <- gradebook::read_gs(input$upload_gs$datapath)
+            uploaded_data <- read_csv(input$upload_gs$datapath)
             data(uploaded_data)
         }, error = function(e) {
             showNotification('Please upload a file with the Gradescope format','',type = "error")
             
         })
     })
+
```

Comment:
Is there a reason that this isn't just deleted? Keep in mind with git, it's very easy to access this code from previous commits if we want to re-incorporate it.